packages/@ember/helper/type-tests/invoke-helper.test.ts

```diff
@@ -14,6 +15,9 @@ class PlusOne extends Helper {
 }
 
 export default class PlusOneComponent extends Component {
+  // Glint would help with this
+  declare args: { number: number };
```

Comment:
This is for EmberComponent, right? Maybe just make some nonsense here that isn't `args` to avoid future confusion?

Reply:
Ah yeah, that's right. I always get them mixed up 😬